packages/renderer/src/lib/dashboard/ProviderConfigured.svelte

```diff
@@ -39,7 +39,13 @@ async function runProvider() {
   runInProgress = false;
 }
 
-onMount(() => {
+onMount(async () => {
+  // Get the autostart configuration setting. If it's disabled, we do *not* want to start the current container provider.
+  const autostart = await window.getConfigurationValue<boolean>('preferences.engine.autostart');
+  if (!autostart) {
+    runAtStart = false;
```

Comment:
This break the original behavior though.
Let's say you have the autostart setting set to false and you are installing podman (idk for what reason this could happen). After the installation complete, you click on `Initialize and start`. The podman machine is initialized but it's not started bc the setting is false.
Shouldn't the click have priority over a setting?

Reply:
You're right. I've changed it to `if (runAtStart && autostart)` instead so that it'll *only* run runProvider() if autostart is set to enabled / defined.

So that means after you press Initialize and start, it'll actually start.

Reply:
Doesn't it have the same outcome? If autostart is false (bc the setting is false) and you click on `Initialize and start`, it will not get in the if branch. I guess the problem is that the `runAtStart` is true by default because the value of `initializationContext.mode`  is `InitializeAndStartMode` even when the initialize and start is not actually clicked.
I mean it is set differently only when the `InitializeMode` is clicked. Maybe it could be set at some other value when the user doesn't initialize the provider but only open up PD 🤔